submit/controllers/verify_user.py

```diff
@@ -0,0 +1,68 @@
+"""
+Controller for verify_user action.
+
+Creates an event of type `core.events.event.VerifyContactInformation`
+"""
+
+from typing import Tuple, Dict, Any, Optional
+
+from wtforms import Form, BooleanField
+from wtforms.validators import InputRequired
+
+from arxiv import status
+from arxiv.base import logging
+import events
+
+logger = logging.getLogger(__name__) #pylint: disable=C0103
+
+Response = Tuple[Dict[str, Any], int, Dict[str, Any]] #pylint: disable=C0103
+
+
+def verify_user(request_params: dict, submission_id: Optional[int]) -> Response:
+    """
+    Converts the verify_user form data and converts it to a
+    `VerifyContactInfromation` event.
+
+    If a submission has not yet been created, the `CreateSubmission` event is
+    raised to yield a submission_id.
+    """
+    form = VerifyUserForm(request_params)
+
+    # Process event if go to next page
+    if request_params.get('action') == 'next' and form.validate():
+        # TODO: Create a concrete User event from cookie info.
+        submitter = events.domain.User(1, email='ian413@cornell.edu',
+                                       forename='Ima', surname='Nauthor')
+
+        # Create submission if it does not yet exist
+        if submission_id is None:
+            submission, _ = events.save(
```

Comment:
The big thing missing here is exception handling. What will you do if the event is not valid? How will you present that information to the user?

Reply:
It's fine if you don't know the answer yet, but I'd at least mock up something minimal so that we can see where it goes. E.g. try-catch block, set [non-field errors on the form](https://github.com/wtforms/wtforms/blob/master/wtforms/form.py#L146).